### PR TITLE
Optimize images, tweak admin, launch feed

### PR DESCRIPTION
**Pull request by mwickett**
It's a little all over the place, but just tidying up before launch.

### CODE REVIEW

File: components/nav/index.tsx [hunk 1]
```diff
@@ -100,6 +100,7 @@ const MenuLinks = ({ isOpen }) => {
         <MenuItem to="/">Home</MenuItem>
         <MenuItem to="/about">About Us</MenuItem>
         <MenuItem to="/faq">FAQ</MenuItem>
+        <MenuItem to="https://kvmhxg5ojy6.typeform.com/to/gUsoYkft">Media</MenuItem>
```
Comment (mwickett) [inline components/nav/index.tsx:103]: Media link to typeform in nav bar

File: pages/_admin/index.tsx [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
           type="button"
           data-id={id}
           data-type="deleted"
-          onClick={(e) => {if(window.confirm('Are you sure you want to delete this?')){updateStory(e)}}}
+          onClick={(e) => {
+            if (window.confirm('Are you sure you want to delete this?')) {
+              updateStory(e)
+            }
+          }}
         >
```
Comment (mwickett) [inline pages/_admin/index.tsx:34]: Show mods if a submission has consented to contact, and include the story ID (and link to what would be the live link if approved)

File: pages/_admin/index.tsx [hunk 3]
```diff
@@ -42,69 +49,69 @@ function Story({ id, title, content, name, postal, ...rest}) {
...
+            <Stack spacing={8}>
+              {stories.map((story) => (
+                <Story key={story.id} {...story} />
+              ))}
+            </Stack>
+          </Container>
+        </>
+      )}
+    </>
+  )
```
Comment (mwickett) [inline pages/_admin/index.tsx:69]: @silent1mezzo Just formatting tidy-up - you should install prettier and eslint plugins :)

Comment (silent1mezzo) [reply pages/_admin/index.tsx:69]: Hmm, I have. Will figure this out

**Review by davesteinberg — CHANGES_REQUESTED** [review]
Nice job on the images, it's so much smoother on my phone.

One thing: you missed changing a '/feed' to '/' on line 16 of `[id].tsx`. As a result, go directly to a story page (e.g. https://www-8ohknf7ku-my-covid-story.vercel.app/story/cknqtf0oz1386cn5fz62eydta) and click the close button, you'll get a 404.

Aside: why did I call it cancel instead of close? If you wanted to change `handleCancel`/`onCancel` to `handleClose`/`onClose` (in `[id].tsx` and `StoryDetail.tsx`), I would not object.😁

**Comment by mwickett** [comment]
oh great catch! I missed that. Fixes for your comments are addressed in my latest commit.

**Review by silent1mezzo — APPROVED** [review]
👍 appreciate the admin cleanup. LGTM

**Review by davesteinberg — COMMENTED** [review]
Changes look good! You can merge PR #71 into this one to pick up my fix for the Other category label.

**Review by davesteinberg — APPROVED** [review]

**Pull request merged**